samples/broker-trigger/README.md

```diff
@@ -2,10 +2,10 @@
 
 ## Prerequisites
 
-install the Broker running:
-`kubectl apply -f https://github.com/knative-sandbox/eventing-rabbitmq/releases/latest/download/rabbitmq-broker.yaml`
+Same as listed [here](../../broker/README.md#prerequisites)
 
-or as per [here](../../broker/README.md)
+Now install the Broker running:
```

Comment:
The previous step points to the [installation](https://github.com/gabo1208/eventing-rabbitmq/blob/broker-and-source-docs-improvements/broker/operator-based.md#installation) which would include installing the rabbitmq-broker.yaml
Is this step required ?